tests/test_features/test_pcoords.py

```diff
@@ -131,3 +157,65 @@ def test_integrated_pcoords(self):
         visualizer.fit_transform(X, y)
         visualizer.poof()
         self.assert_images_similar(visualizer)
+
+    @staticmethod
+    def test_static_subsample():
+        """
+        Assert output of subsampling method against expectations
+        """
+
+        ntotal = 100
+        ncols = 50
+
+        y = np.arange(ntotal).reshape(ntotal, 1)
+        X = np.ones((ntotal, ncols)) * y
```

Comment:
I very much like this idea for a test case, it definitely allows you to determine the ordering with ease! Note, however, that `y` should have shape `(100,)` and instead, it has shape `(100,1)`. Because you need this shape in order to compute `X`, I recommend the following:

```python
y = np.arange(ntotal)
X = np.ones((ntotal, ncols)) * y.reshape(ntotal,1) 
```

Reply:
Good call. Now done.